errors/failed-loading-swc.md

```diff
@@ -10,6 +10,8 @@ SWC requires a binary be downloaded that is compatible specific to your system.
 
 When on an M1 Mac and switching from a Node.js version without M1 support e.g. v14 to a version with e.g. v16, you may need a different swc dependency which can require re-installing `node_modules` (`npm i --force` or `yarn install --force`).
 
+On Windows make sure you have Microsoft Visual C++ Redistributable installed. https://aka.ms/vs/17/release/vc_redist.x64.exe
```

Comment:
```suggestion
On Windows make sure you have Microsoft Visual C++ Redistributable installed. https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist
```
Let's link to the docs instead of the binary directly in case a different CPU arch is needed and it explains more what they are downloading.

Reply:
Make sense, done.